.env.example

```diff
@@ -1,6 +1,7 @@
 DATABASE_URL='postgresql://<user>:<pass>@<db-host>:<db-port>/<db-name>'
 GOOGLE_API_CREDENTIALS='secret'
 BASE_URL='http://localhost:3000'
+NEXT_PUBLIC_BASE_URL=http://localhost:3000
```

Comment:
I don't understand the reasoning for this new variable, why can't we just use `BASE_URL`?

Reply:
Hi, its because `BASE_URL` is not visible to the browser (and we need it for the iframe link). In order to get it visible to the browser, we need prefixing a variable with `NEXT_PUBLIC`. You can reed more about here:

https://nextjs.org/docs/basic-features/environment-variables